doc/building.rst

```diff
@@ -66,11 +66,18 @@ recipes.
 
 To set up your build environment for development using HomeBrew_:
 
-1. Add the ownCloud repository using the following command::
+1. Install Xcode
+2. Install Xcode command line tools::
+    xcode-select --install
+
+3. Install homebrew::
+    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
```

Comment:
You need an empty line between 'Install homebrew::'  and the command for the markup to render the code blocks correctly.